inginious/frontend/pages/course_admin/task_edit.py

```diff
@@ -142,6 +142,8 @@ def POST_AUTH(self, courseid, taskid):  # pylint: disable=arguments-differ
 
             # Network grading
             data["network_grading"] = "network_grading" in data
+
```

Comment:
This is a bit pedantic but to keep PRs the cleanest and easiest to review, please avoid to commit files with no useful changes.

I strongly recommend you to give a first review of your PR by yourself before submitting it for review. This is also useful to when debugging lines remain: there is no chance the PR will be merged in those cases so that automatically requires a second review.